Energy Saver/Pages/Delete.cshtml.cs

```diff
@@ -33,24 +33,32 @@ public async Task<IActionResult> OnGetAsync(int? id)
         {
             if (id == null || _context.Taxes == null)
             {
-                return RedirectToPage("./Index");
```

Comment:
You suggested me to delete NotFound() references, you should too ;)

Reply:
I don't think we should get rid of the not found all together, it's good in some scenarios.